File: chatwoot/java/src/main/java/oseg/chatwoot_examples/DeleteAgentInTeamExample.java
package oseg.chatwoot_examples;

import com.chatwoot.client.ApiException;
import com.chatwoot.client.Configuration;
import com.chatwoot.client.api.*;
import com.chatwoot.client.auth.*;
import com.chatwoot.client.JSON;
import com.chatwoot.client.model.*;

import java.io.File;
import java.time.LocalDate;
import java.time.OffsetDateTime;
import java.util.ArrayList;
import java.util.List;
import java.util.Map;

public class DeleteAgentInTeamExample
{
    public static void main(String[] args)
    {
        var config = Configuration.getDefaultApiClient();
        config.setApiKey("USER_API_KEY");

        var deleteAgentInTeamRequest = new DeleteAgentInTeamRequest();
        deleteAgentInTeamRequest.userIds(List.of ());

        try
        {
            new TeamsApi(config).deleteAgentInTeam(
                null, // accountId
                null, // teamId
                deleteAgentInTeamRequest // data
            );
        } catch (ApiException e) {
            System.err.println("Exception when calling TeamsApi#deleteAgentInTeam");
            System.err.println("Status code: " + e.getCode());
            System.err.println("Reason: " + e.getResponseBody());
            System.err.println("Response headers: " + e.getResponseHeaders());
            e.printStackTrace();
        }
    }
}

File: chatwoot/kotlin/src/main/kotlin/oseg/chatwoot_examples/DeleteAgentInTeamExample.kt
package oseg.chatwoot_examples

import com.chatwoot.client.infrastructure.*
import com.chatwoot.client.apis.*
import com.chatwoot.client.models.*

import java.io.File
import java.time.LocalDate
import java.time.OffsetDateTime
import kotlin.collections.ArrayList
import kotlin.collections.List
import kotlin.collections.Map
import com.squareup.moshi.adapter

@ExperimentalStdlibApi
class DeleteAgentInTeamExample
{
    fun deleteAgentInTeam()
    {
        ApiClient.apiKey["api_access_token"] = "USER_API_KEY"

        val deleteAgentInTeamRequest = DeleteAgentInTeamRequest(
            userIds = listOf (),
        )

        try
        {
            TeamsApi().deleteAgentInTeam(
                accountId = 0,
                teamId = 0,
                _data = deleteAgentInTeamRequest,
            )
        } catch (e: ClientException) {
            println("4xx response calling TeamsApi#deleteAgentInTeam")
            e.printStackTrace()
        } catch (e: ServerException) {
            println("5xx response calling TeamsApi#deleteAgentInTeam")
            e.printStackTrace()
        }
    }
}
